stranger/utils.py

```diff
@@ -158,6 +158,28 @@ def get_exhu_repeat_res_from_alts(variant_info: dict):
             raise SyntaxError("Allele on wrong format")
     return repeat_res
 
+def get_repeat_id(variant_info):
+    """
+    First tries to get variant id from REPID,
+    if that is not sucessful, try to get variant id from TRID (TRGT).
+    If the ID is formatted with underscore (STRchive),
+    grab the part which is after the underscore, otherwise take the whole ID (PacBio).
+    """
+    info_dict = variant_info.get('info_dict', {})
+    
+    repid = info_dict.get('REPID')
+    
+    if repid:
+        return repid
+    else:
```

Comment:
```suggestion
trid = info_dict.get('TRID')
if not trid:
    return None    
if '_' in trid:
    return trid.split('_', 1)[1]
return trid
```

Reply:
Hihi, thank you, and let's also flatten it for that apparent complexity gain. Not being serious about this being needed here, just showing a coding style it took me some time to adopt when arriving from C/C++/perl. 😊